backends/windows/kernelspace/sink.go

```diff
@@ -141,20 +158,30 @@ func (s *Backend) Setup() {
 		15*time.Second,
 		"sync period duration")
 
-	klog.V(0).InfoS("Using Windows Kernel Proxier. node ip  %v", nodeip)
+	klog.Info("Starting Windows Kernel Proxier.")
+	klog.InfoS("  Cluster CIDR", "clusterCIDR", *clusterCIDR)
```

Comment:
should this be on (5) or debug level?

Reply:
I'm generally a fan of logging parameters at max verbosity but up to you